### DISCUSSION TITLE
Control fan speed of GPU

### DISCUSSION THREAD

**Discussion by whoozy82** [Rem0o/FanControl.Releases#351] [Q&A] [answered]
Hi.
I am using Fan Speed to control CPU and chassis fans.
The source is a mix of CPU, GPU and chassis sensors.
This works great. The only thing that does not work is the control of the GPU fan speed.
I have a RX 6600 XT an it works well with the Adrenalin software.
Maybe this software overrules Fan Speed? Or is there something else?

Thanks
Kay

BTW: Fan Speed is amazing! Great work!

**Comment by Rem0o** [comment, marked as answer]
The backend I'm using does not support XT cards very well, if at all AFAIK. AMD third party API support is horrendous at best.
